src/index.css

```diff
@@ -69,38 +56,559 @@ a {
+  border: none;
+  outline: none;
+  font-weight: 300;
+  color: #4386bf;
 }
 
-.secondCol {
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-  gap: 1rem;
+.divider {
+  width: 0.125rem;
+  height: 1.2rem;
+  margin: 0 5% 0 5%;
+  border: solid 1px var(--white-three);
+}
+
+.hrDivider {
+  width: 100%;
+  height: 0.125rem;
+  margin: 2.5rem 0 2.5rem 0;
+  border: solid 1px var(--white-three);
 }
 
-.progressBarAndUpdateButton {
+.bookForm {
   display: flex;
   flex-direction: row;
+  margin-bottom: 3rem;
   justify-content: space-between;
+  width: 100%;
+  padding-top: 2rem;
+  padding-bottom: 4rem;
 }
 
-.progressImg {
-  max-width: 5rem;
-  max-height: 5rem;
+input[type='text'],
+select {
+  border: solid 1px var(--white-three);
+  background-color: var(--white-two);
+  padding-left: 1rem;
+  outline: none;
 }
 
-.bookForm {
+.categoryPage {
   display: flex;
-  flex-direction: row;
-  justify-content: space-between;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 1rem 0 3rem 0;
+}
+
+.addBook {
+  min-width: 10vw;
+  height: 2.5rem;
+  border-radius: 4px;
+  font-family: 'Roboto Slab', serif;
+  background-color: var(--primary-color);
+  outline: none;
+  border: solid 1px var(--white-three);
+  color: white;
+}
+
+/* mobile version */
+@media (width >= 765px) {
+  .navItems {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    gap: 5%;
+  }
+
+  .panel-bg {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    width: 100%;
+    min-height: 5.938rem;
+    background-color: white;
+    position: sticky;
+    padding: 0 2.5rem 0 2.5rem;
+    top: 0;
+    overflow-x: hidden;
+  }
+
+  .bookstore-CMS {
+    font-family: 'Montserrat', sans-serif;
+    font-size: 32px;
+    font-weight: 600;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
+    letter-spacing: normal;
+    color: #0290ff;
+  }
+
+  .books-nav {
+    color: #121212;
+    font-family: 'Montserrat', sans-serif;
+    font-size: 17px;
+    font-weight: normal;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
+    letter-spacing: 1.9px;
+  }
+
+  .pageWrapper {
+    margin: auto;
+    width: 80vw;
+    background-color: #fafafa;
+    border-radius: 4px;
+    box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.2);
+  }
+
+  .innerWrapper {
+    width: 90%;
+    margin: auto;
+  }
+
+  .bookPanelBg {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: 100%;
+    margin-top: 4.375rem;
+    gap: 2rem;
+  }
+
+  .bookPanel {
+    min-height: 11rem;
+    padding: 1rem 2.5rem 1rem 2.5rem;
+    border-radius: 4px;
+    border: solid 1px var(--white-three);
+    background-color: var(--white-two);
+    align-items: center;
+  }
+
+  .bookDetails {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+  }
```

Comment:
- [ ] Kindly ensure that your design matches the Zeplin design to meet the requirements  👍 

### Your design

![Capture](https://github.com/Rafiduzzaman/bookstore/assets/80458199/360189c3-2846-498b-aafa-5d5811432a37)

### Zeplin design

![Capture0](https://github.com/Rafiduzzaman/bookstore/assets/80458199/7f920332-0009-4c5e-9b24-bbacb3618c62)